posts/2021-02-15-eclipse-transformer.adoc

```diff
@@ -0,0 +1,311 @@
+
+Here is a table of features which use the same short names and for which only the feature version is updated:
+
+.Java / Jakarta EE to Jakarta EE 9 Feature Updates: Version Only
+|===
+| Java / Jakarta EE feature name | Jakarta EE 9 feature name
+
+| appClientSupport-1.0
+| appClientSupport-2.0
+
+| appSecurity-3.0
+| appSecurity-4.0
+
+| batch-1.0
+| batch-2.0
+
+| beanValidation-2.0
+| beanValidation-3.0
+
+| cdi-2.0
+| cdi-3.0
+
+| concurrent-1.0
+| concurrent-2.0
+
+| jsonb-1.0
+| jsonb-2.0
+
+| jsonbContainer-1.0
+| jsonbContainer-2.0
+
+| jsonp-1.1
+| jsonp-2.0
+
+| jsonpContainer-1.1
+| jsonpContainer-2.0
+
+| managedBeans-1.0
+| managedBeans-2.0
+
+| mdb-3.2
+| mdb-4.0
+
+| servlet-4.0
+| servlet-5.0
+
+| webProfile-8.0
+| webProfile-9.0
+
+| websocket-1.1
+| websocket-2.0
+|===
+
+Here is a table of features which update both the feature short name and the feature version.
+
+For these cases, if the incorrect Java EE feature short name is used with a new Jakarta EE 9 version, for example, changing "ejb-3.2" to "ejb-4.0", the server will not start, and the server will provide information to say that an updated feature name must be used.
+
+.Java / Jakarta EE to Jakarta EE 9 Feature Updates: Short Name and Version
+|===
+| Java / Jakarta EE feature name | Jakarta EE 9 feature name
+
+| javaee-8.0
+| jakartaee-9.0
+
+| javaeeClient-8.0
+| jakartaeeClient-9.0
+
+| ejb-3.2
+| enterpriseBeans-4.0
+
+| ejbHome-3.2
+| enterpriseBeansHome-4.0
+
+| ejbLite-3.2
+| enterpriseBeansLite-4.0
+
+| ejbPersistentTimer-3.2
+| enterpriseBeansPersistentTimer-4.0
+
+| ejbRemote-3.2
+| enterpriseBeansRemote-4.0
+
+| el-3.0
+| expressionLanguage-4.0
+
+| jacc-1.5
+| appAuthorization-2.0
+
+| jaspic-1.1
+| appAuthentication-2.0
+
+| javaMail-1.6
+| mail-2.0
+
+| jaxb-2.2
+| xmlBinding-3.0
+
+| jaxrs-2.1
+| restfulWS-3.0
+
+| jaxrsClient-2.1
+| restfulWSClient-3.0
+
+| jaxws-2.2
+| xmlWS-3.0
+
+| jca-1.7
+| connectors-2.0
+
+| jcaInboundSecurity-1.0
+| connectorsInboundSecurity-2.0
+
+| jms-2.0
+| messaging-3.0
+
+| jpa-2.2
+| persistence-3.0
+
+| jpaContainer-2.x
+| persistenceContainer-3.0
+
+| jsf-2.3
+| faces-3.0
+
+| jsfContainer-2.3
+| facesContainer-3.0
+
+| jsp-2.3
+| pages-3.0
+
+| wasJmsClient-2.0
+| messagingClient-3.0
+
+| wasJmsSecurity-1.0
+| messagingSecurity-3.0
+
+| wasJmsServer-1.0
+| messagingServer-3.0
+|===
+
+== Reference resources
+
+=== Eclipse Transformer project links
+
+Main Eclipse Transformer project page: link:https://projects.eclipse.org/projects/technology.transformer[Eclipse Transformer].
+
+A direct link to the transformer GIT repository: link:https://github.com/OpenLiberty/open-liberty/tree/integration/dev/wlp-jakartaee-transform[dev/wlp-jakartaee-transform].
```

Comment:
This is the wrong link and the wrong text for the link.  It should point to eclipse repo, not the Open Liberty repo